lib/garage/optional_response_body_responder.rb

```diff
@@ -3,6 +3,8 @@ module Garage::OptionalResponseBodyResponder
 
   def api_behavior(*)
     case
+    when patch? && options[:patch] && options[:patch][:body]
+      display resource, status: options[:patch][:status] || :ok
```

Comment:
oops... Sorry, I was seeing a [v1.5.2](https://github.com/cookpad/garage/tree/v1.5.2). It will only add a test case.